cli/azd/pkg/input/console.go

```diff
@@ -17,11 +17,32 @@ import (
 	"github.com/azure/azure-dev/cli/azd/pkg/contracts"
 	"github.com/azure/azure-dev/cli/azd/pkg/output"
 	"github.com/mattn/go-colorable"
+	"github.com/theckman/yacspin"
+)
+
+type MessageUxType int
+
+const (
+	Title MessageUxType = iota
+	Step
+	StepDone
+	StepFailed
+	ResultSuccess
```

Comment:
Is there a reason to share these enumerations as a single `type`? `Title`, `ResultSuccess`, and `ResultFailure` seem to be used only when formatting text, while the others are used for displaying progress

Reply:
That's to keep just one `Message()` function.

Reply:
@vhvb1989 feel free to push back, but I think the last time we discussed this, separating out the enum types creates type-safety around formatting message options versus spinner options. My main concern is to reduce avoidable runtime errors via type-safety.

Reply:
Updated it.

The other reason why I ended up not doing it is because I talked to Matt and understood a little better approach to use as part of the next PR. So, most likely these types will be gone anyway